src/main/java/com/timelogr/enterprise/Client.java

```diff
@@ -0,0 +1,23 @@
+package com.timelogr.enterprise;
+
+import lombok.Data;
+
+public @Data
+class Client {
+
+    private long clientID;
+    private String clientName;
+    private String clientEmail;
+    private String clientNumber;
+
+    public Client() {
+        super();
+    }
+
+    public Client(long id, String name, String email, String number) {
+        this.clientID = id;
+        this.clientName = name;
+        this.clientEmail = email;
+        this.clientNumber = number;
+    }
+}
```

Comment:
This is for all classes that I changed to follow the naming conventions: 
- I went ahead and installed the Lombok dependency that you can use so that you don't need to manually generate/type getters and setters. This may save you some time.
- I also went ahead and changed up your classes' variable names to be more descriptive (though it may be not needed)

Reply:
Good.

One suggestion: since the class name is Client, no need to repeat that word on each of the attributes (clientID, clientName, etc).  That is redundant.